docs/Design/Enums/Enum Declarations.md

```diff
@@ -88,6 +88,16 @@ enum Direction {
 }
 ```
 
+### JavaScript
+
+```
+enum Direction {
+    Unknown = 0,
+    Horizontal = 1,
+    Vertical = 2
+}
```

Comment:
JavaScript doesn't have enums the same way TypeScript does. It just makes a variable.

You can check how they look with aka.ms/gls-demo.